examples/mars-theme-example/package.json

```diff
@@ -24,7 +24,6 @@
     "@frontity/mars-theme": "^1.3.1",
     "@frontity/tiny-router": "^1.1.0",
     "@frontity/wp-source": "^1.5.0",
-    "babel-plugin-frontity": "^1.0.1",
```

Comment:
@luisherranz Noticed that our plugin is a dependency in all of the examples. Should I remove the dependency from the packages?

Also how should we deprecate the plugin? Remove it entirely?

Reply:
> Should I remove the dependency from the packages?

That's weird. That dependency should not be there, it makes no sense.

> Also how should we deprecate the plugin? Remove it entirely?

Nah, I don't think so. Nobody uses it, only `@frontity/core` internally. And I think that if we deprecate it, it will throw a warning for people still on old `@frontity/core` versions.

Reply:
Yes, I don't know why it's there. It should only be a dependency of `@frontity/core`.

Reply:
Ok, I see Cristian has already done so in https://github.com/frontity/frontity/pull/666/commits/979c44edcbe9b6615f1a74236bc2f350e9c7c0d4.